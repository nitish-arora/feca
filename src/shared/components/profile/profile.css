.ne-s-profile-container {
    height: 70px;
    float: right;
}

.ne-s-profile-container .profile-toolbar {
    padding: 0 20px;
    display: flex;
    align-items: center;
    height: 70px;
}

.ne-s-profile-container .profile-toolbar .ne-s-text-name {
    margin-right: 32px;
    margin-left: 5px;
    font-size: 14px;
    height: 24px;
}

.ne-s-profile-container .profile-toolbar .ne-s-logout-btn {
    border: none;
    background: transparent;
    cursor: pointer;
    font-size: 16px;
    padding: 5px 7px;
}