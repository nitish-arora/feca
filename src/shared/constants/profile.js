export const ACCOUNT = "Account";
export const LOGOUT = "Logout";