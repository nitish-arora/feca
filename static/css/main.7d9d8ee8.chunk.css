body{margin:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen,Ubuntu,Cantarell,Fira Sans,Droid Sans,Helvetica Neue,sans-serif;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}code{font-family:source-code-pro,Menlo,Monaco,Consolas,Courier New,monospace}.App{text-align:center}.App-logo{-webkit-animation:App-logo-spin 20s linear infinite;animation:App-logo-spin 20s linear infinite;height:40vmin;pointer-events:none}.App-header{background-color:#282c34;min-height:100vh;display:flex;flex-direction:column;align-items:center;justify-content:center;font-size:calc(10px + 2vmin);color:#fff}.App-link{color:#61dafb}@-webkit-keyframes App-logo-spin{0%{-webkit-transform:rotate(0deg);transform:rotate(0deg)}to{-webkit-transform:rotate(1turn);transform:rotate(1turn)}}@keyframes App-logo-spin{0%{-webkit-transform:rotate(0deg);transform:rotate(0deg)}to{-webkit-transform:rotate(1turn);transform:rotate(1turn)}}.ne-s-login-form-container{position:fixed;top:0;left:0;width:100%;height:100%;display:flex;justify-content:center;align-items:center;background-color:transparent}.ne-s-login-form-container .ne-s-login-form{width:400px;position:relative;text-align:right;background:#fff}.ne-s-login-form .ne-s-login-form-toolbar{display:flex;box-sizing:border-box;padding:0 16px;width:100%;flex-direction:row;align-items:center;white-space:nowrap;background-color:#68d122!important;color:#fff!important;height:64px}.ne-s-login-form-toolbar>h2{font-weight:500}.ne-s-login-form form{width:270px!important;margin:20px 32px 0}.ne-s-login-form button{margin:10px 32px 32px;border-radius:30px;height:36px;width:88px;font-weight:700;text-transform:uppercase}.ne-s-login-form .error{height:19px;width:100%;text-align:center;padding-bottom:10px;color:#d63a47}.ne-s-login-form button.disabled{background-color:#f2f2f2;color:#b6b6b6}.ne-s-login-form button.not-disabled{background-color:#6aa81c;color:#fff}.ne-s-login-component-container{position:fixed;top:0;left:0;width:100%;height:100%;display:flex;justify-content:center;align-items:center}.ne-s-login-component-container .ne-s-logotype-item{width:100%;height:100vh;min-height:700px;overflow:hidden;position:relative;background:url(/feca/static/media/hero-startpage-better-gaming-1920x1000.bfd50cf9.jpg) no-repeat 50%;background-size:cover}.ne-s-login-component-container .ne-s-logotype-image{position:absolute;max-width:110px;top:20px;left:100px}.ne-s-profile-container{height:70px;float:right}.ne-s-profile-container .profile-toolbar{padding:0 20px;display:flex;align-items:center;height:70px}.ne-s-profile-container .profile-toolbar .ne-s-text-name{margin-right:32px;margin-left:5px;font-size:14px;height:24px}.ne-s-profile-container .profile-toolbar .ne-s-logout-btn{border:none;background:transparent;cursor:pointer;font-size:16px;padding:5px 7px}.ne-s-sidenav-container{height:100vh;background:#212426;min-width:256px}.ne-s-sidenav-container .ne-s-logotype-item{height:70px;display:flex;justify-content:center}.ne-s-logotype-item>img{max-width:112px}.ne-s-tree .menu-list{margin:0;text-align:left;font-size:13px;min-height:35px;padding:10px 20px;cursor:pointer}.menu-list .text-header{font-size:13px;font-weight:700;text-transform:uppercase;position:absolute;left:70px}.ne-s-tree .menu-list{color:#b6b6b6;display:flex;align-items:center}.ne-s-tree .selected-sidenav{color:#68d122}.ne-s-main-container{background:#f2f2f2;height:100vh;display:flex}.ne-s-main-container .ne-s-target-toolbar{height:70px;background:#fff}.ne-s-main-container .ne-s-sidenav-target-container{height:100%;width:100%}.ne-s-sidenav-target-container .ne-s-sidenav-outlet>h3{padding-left:16px}.ne-s-sidenav-target-container .ne-s-sidenav-outlet{width:100%;height:calc(100% - 74px)}.ne-s-sidenav-outlet .home-title{padding:20px}.ne-s-logout-container{position:fixed;top:0;left:0;width:100%;height:100%;display:flex;justify-content:center;align-items:center;background-color:#17191a}.ne-s-logout-container .ne-s-logotype-item{position:absolute;max-width:110px;top:20px;left:100px;display:flex;align-items:center}.ne-s-logotype-item img{width:110px}.ne-s-logotype-item span{margin-left:15px}.ne-s-logout-container .ne-s-logout-page{position:relative;text-align:right;background-color:#212426;color:#fff!important}.ne-s-logout-page>h2{font-size:32px;line-height:36px;margin:20px 32px 0}.ne-s-logout-page>h3{font-size:14px;font-style:italic;line-height:36px;margin:0 32px 20px}.ne-s-logout-page>h3,.ne-s-logout-page h2{text-align:left;font-weight:500}.ne-s-logout-page>.ne-s-netent-button{margin:0 32px 32px;min-width:120px;border-radius:30px;background-color:#68d122;color:#fff;text-transform:uppercase;font-weight:600}.ne-s-logout-page .ne-s-logout-toolbar{background-color:#68d122!important;color:#fff!important;height:64px;display:flex;box-sizing:border-box;padding:0 16px;width:100%;flex-direction:row;align-items:center;white-space:nowrap}
/*# sourceMappingURL=main.7d9d8ee8.chunk.css.map */