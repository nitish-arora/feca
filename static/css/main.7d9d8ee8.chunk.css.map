{"version":3,"sources":["index.css","App.css","login.css","login-view.css","profile.css","sidenav.css","dashboard.css","logout.css"],"names":[],"mappings":"AAAA,KACE,QAAS,CACT,mIAEY,CACZ,kCAAmC,CACnC,iCACF,CAEA,KACE,uEAEF,CCZA,KACE,iBACF,CAEA,UACE,mDAA4C,CAA5C,2CAA4C,CAC5C,aAAc,CACd,mBACF,CAEA,YACE,wBAAyB,CACzB,gBAAiB,CACjB,YAAa,CACb,qBAAsB,CACtB,kBAAmB,CACnB,sBAAuB,CACvB,4BAA6B,CAC7B,UACF,CAEA,UACE,aACF,CAEA,iCACE,GACE,8BAAuB,CAAvB,sBACF,CACA,GACE,+BAAyB,CAAzB,uBACF,CACF,CAPA,yBACE,GACE,8BAAuB,CAAvB,sBACF,CACA,GACE,+BAAyB,CAAzB,uBACF,CACF,CChCA,2BACI,cAAe,CACf,KAAM,CACN,MAAO,CACP,UAAW,CACX,WAAY,CACZ,YAAa,CACb,sBAAuB,CACvB,kBAAmB,CACnB,4BACJ,CAEA,4CACI,WAAY,CACZ,iBAAkB,CAClB,gBAAiB,CACjB,eACJ,CAEA,0CACI,YAAa,CACb,qBAAsB,CACtB,cAAe,CACf,UAAW,CACX,kBAAmB,CACnB,kBAAmB,CACnB,kBAAmB,CACnB,kCAAoC,CACpC,oBAAuB,CACvB,WACA,CAEJ,4BACI,eACJ,CAEA,sBACI,qBAAuB,CACvB,kBACJ,CAEA,wBACI,qBAA2B,CAC3B,kBAAmB,CACnB,WAAY,CACZ,UAAW,CACX,eAAiB,CACjB,wBACJ,CAEA,wBACI,WAAY,CACZ,UAAW,CACX,iBAAkB,CAClB,mBAAoB,CACpB,aACJ,CAEA,iCACI,wBAAyB,CACzB,aACJ,CAEA,qCACI,wBAAyB,CACzB,UACJ,CClEA,gCACI,cAAe,CACf,KAAM,CACN,MAAO,CACP,UAAW,CACX,WAAY,CACZ,YAAa,CACb,sBAAuB,CACvB,kBACJ,CAEA,oDACI,UAAW,CACX,YAAa,CACb,gBAAiB,CACjB,eAAgB,CAChB,iBAAkB,CAClB,oGAAkG,CAClG,qBACJ,CAEA,qDACI,iBAAkB,CAClB,eAAgB,CAChB,QAAS,CACT,UACJ,CC1BA,wBACI,WAAY,CACZ,WACJ,CAEA,yCACI,cAAe,CACf,YAAa,CACb,kBAAmB,CACnB,WACJ,CAEA,yDACI,iBAAkB,CAClB,eAAgB,CAChB,cAAe,CACf,WACJ,CAEA,0DACI,WAAY,CACZ,sBAAuB,CACvB,cAAe,CACf,cAAe,CACf,eACJ,CCzBA,wBACI,YAAa,CACb,kBAAmB,CACnB,eACJ,CAEA,4CACI,WAAY,CACZ,YAAa,CACb,sBACJ,CAEA,wBACI,eACJ,CAEA,sBACI,QAAS,CAET,eAAgB,CAChB,cAAe,CACf,eAAgB,CAChB,iBAAkB,CAClB,cACJ,CAEA,wBACI,cAAe,CACf,eAAiB,CACjB,wBAAyB,CACzB,iBAAkB,CAClB,SACJ,CAEA,sBACI,aAAc,CACd,YAAa,CACb,kBACJ,CAEA,6BACI,aACJ,CC1CA,qBACI,kBAAmB,CACnB,YAAa,CACb,YACJ,CAEA,0CACI,WAAY,CACZ,eACJ,CAEA,oDACI,WAAY,CACZ,UACJ,CAEA,uDACI,iBACJ,CAEA,oDACI,UAAW,CACX,wBACJ,CAEA,iCACI,YACJ,CC3BA,uBACI,cAAe,CACf,KAAM,CACN,MAAO,CACP,UAAW,CACX,WAAY,CACZ,YAAa,CACb,sBAAuB,CACvB,kBAAmB,CACnB,wBACJ,CAEA,2CACI,iBAAkB,CAClB,eAAgB,CAChB,QAAS,CACT,UAAW,CACX,YAAa,CACb,kBACJ,CAEA,wBACI,WACJ,CAEA,yBACI,gBACJ,CAEA,yCACI,iBAAkB,CAClB,gBAAiB,CACjB,wBAAyB,CACzB,oBACJ,CAEA,qBACI,cAAe,CACf,gBAAiB,CACjB,kBACJ,CAEA,qBACI,cAAe,CACf,iBAAkB,CAClB,gBAAiB,CACjB,kBACJ,CAEA,0CACI,eAAgB,CAChB,eACJ,CAEA,sCACI,kBAAwB,CACxB,eAAgB,CAChB,kBAAmB,CACnB,wBAAyB,CACzB,UAAY,CACZ,wBAAyB,CACzB,eACJ,CAEA,uCACI,kCAAoC,CACpC,oBAAuB,CACvB,WAAY,CACZ,YAAa,CACb,qBAAsB,CACtB,cAAe,CACf,UAAW,CACX,kBAAmB,CACnB,kBAAmB,CACnB,kBACJ","file":"main.7d9d8ee8.chunk.css","sourcesContent":["body {\n  margin: 0;\n  font-family: -apple-system, BlinkMacSystemFont, \"Segoe UI\", \"Roboto\", \"Oxygen\",\n    \"Ubuntu\", \"Cantarell\", \"Fira Sans\", \"Droid Sans\", \"Helvetica Neue\",\n    sans-serif;\n  -webkit-font-smoothing: antialiased;\n  -moz-osx-font-smoothing: grayscale;\n}\n\ncode {\n  font-family: source-code-pro, Menlo, Monaco, Consolas, \"Courier New\",\n    monospace;\n}\n",".App {\n  text-align: center;\n}\n\n.App-logo {\n  animation: App-logo-spin infinite 20s linear;\n  height: 40vmin;\n  pointer-events: none;\n}\n\n.App-header {\n  background-color: #282c34;\n  min-height: 100vh;\n  display: flex;\n  flex-direction: column;\n  align-items: center;\n  justify-content: center;\n  font-size: calc(10px + 2vmin);\n  color: white;\n}\n\n.App-link {\n  color: #61dafb;\n}\n\n@keyframes App-logo-spin {\n  from {\n    transform: rotate(0deg);\n  }\n  to {\n    transform: rotate(360deg);\n  }\n}\n",".ne-s-login-form-container {\r\n    position: fixed;\r\n    top: 0;\r\n    left: 0;\r\n    width: 100%;\r\n    height: 100%;\r\n    display: flex;\r\n    justify-content: center;\r\n    align-items: center;\r\n    background-color: transparent;\r\n}\r\n\r\n.ne-s-login-form-container .ne-s-login-form {\r\n    width: 400px;\r\n    position: relative;\r\n    text-align: right;\r\n    background: white;\r\n}\r\n\r\n.ne-s-login-form .ne-s-login-form-toolbar {\r\n    display: flex;\r\n    box-sizing: border-box;\r\n    padding: 0 16px;\r\n    width: 100%;\r\n    flex-direction: row;\r\n    align-items: center;\r\n    white-space: nowrap;\r\n    background-color: #68d122 !important;\r\n    color: white !important;\r\n    height: 64px;\r\n    }\r\n\r\n.ne-s-login-form-toolbar > h2 {\r\n    font-weight: 500;\r\n}\r\n\r\n.ne-s-login-form form {\r\n    width: 270px !important;\r\n    margin: 20px 32px 0 32px;\r\n}\r\n\r\n.ne-s-login-form button {\r\n    margin: 10px 32px 32px 32px;\r\n    border-radius: 30px;\r\n    height: 36px;\r\n    width: 88px;\r\n    font-weight: bold;\r\n    text-transform: uppercase;\r\n}\r\n\r\n.ne-s-login-form .error {\r\n    height: 19px;\r\n    width: 100%;\r\n    text-align: center;\r\n    padding-bottom: 10px;\r\n    color: #d63a47;\r\n}\r\n\r\n.ne-s-login-form button.disabled {\r\n    background-color: #f2f2f2;\r\n    color: #b6b6b6;\r\n}\r\n\r\n.ne-s-login-form button.not-disabled {\r\n    background-color: #6aa81c;\r\n    color: white;\r\n}\r\n\r\n.ne-s-login-form .form-field {\r\n    /* height: 35px; */\r\n}",".ne-s-login-component-container {\r\n    position: fixed;\r\n    top: 0;\r\n    left: 0;\r\n    width: 100%;\r\n    height: 100%;\r\n    display: flex;\r\n    justify-content: center;\r\n    align-items: center;\r\n}\r\n  \r\n.ne-s-login-component-container .ne-s-logotype-item {\r\n    width: 100%;\r\n    height: 100vh;\r\n    min-height: 700px;\r\n    overflow: hidden;\r\n    position: relative;\r\n    background: url('../../assets/images/hero-startpage-better-gaming-1920x1000.jpg') no-repeat center;\r\n    background-size: cover;\r\n}\r\n  \r\n.ne-s-login-component-container .ne-s-logotype-image {\r\n    position: absolute;\r\n    max-width: 110px;\r\n    top: 20px;\r\n    left: 100px;\r\n}",".ne-s-profile-container {\r\n    height: 70px;\r\n    float: right;\r\n}\r\n\r\n.ne-s-profile-container .profile-toolbar {\r\n    padding: 0 20px;\r\n    display: flex;\r\n    align-items: center;\r\n    height: 70px;\r\n}\r\n\r\n.ne-s-profile-container .profile-toolbar .ne-s-text-name {\r\n    margin-right: 32px;\r\n    margin-left: 5px;\r\n    font-size: 14px;\r\n    height: 24px;\r\n}\r\n\r\n.ne-s-profile-container .profile-toolbar .ne-s-logout-btn {\r\n    border: none;\r\n    background: transparent;\r\n    cursor: pointer;\r\n    font-size: 16px;\r\n    padding: 5px 7px;\r\n}",".ne-s-sidenav-container {\r\n    height: 100vh;\r\n    background: #212426;\r\n    min-width: 256px;\r\n}\r\n\r\n.ne-s-sidenav-container .ne-s-logotype-item {\r\n    height: 70px;\r\n    display: flex;\r\n    justify-content: center;\r\n}\r\n\r\n.ne-s-logotype-item > img {\r\n    max-width: 112px;\r\n}\r\n\r\n.ne-s-tree .menu-list {\r\n    margin: 0;\r\n    padding: 0;\r\n    text-align: left;\r\n    font-size: 13px;\r\n    min-height: 35px; \r\n    padding: 10px 20px;\r\n    cursor: pointer;\r\n}\r\n\r\n.menu-list .text-header {\r\n    font-size: 13px;\r\n    font-weight: bold;\r\n    text-transform: uppercase;\r\n    position: absolute;\r\n    left: 70px;\r\n}\r\n\r\n.ne-s-tree .menu-list {\r\n    color: #b6b6b6;\r\n    display: flex;\r\n    align-items: center;\r\n}\r\n\r\n.ne-s-tree .selected-sidenav {\r\n    color: #68d122;\r\n}",".ne-s-main-container {\r\n    background: #f2f2f2;\r\n    height: 100vh;\r\n    display: flex;\r\n}\r\n\r\n.ne-s-main-container .ne-s-target-toolbar {\r\n    height: 70px;\r\n    background: white;\r\n}\r\n\r\n.ne-s-main-container .ne-s-sidenav-target-container {\r\n    height: 100%;\r\n    width: 100%;\r\n}\r\n\r\n.ne-s-sidenav-target-container .ne-s-sidenav-outlet > h3 {\r\n    padding-left: 16px;\r\n}\r\n\r\n.ne-s-sidenav-target-container .ne-s-sidenav-outlet {\r\n    width: 100%;\r\n    height: calc(100% - 74px);\r\n}\r\n\r\n.ne-s-sidenav-outlet .home-title {\r\n    padding: 20px;\r\n}",".ne-s-logout-container {\r\n    position: fixed;\r\n    top: 0;\r\n    left: 0;\r\n    width: 100%;\r\n    height: 100%;\r\n    display: flex;\r\n    justify-content: center;\r\n    align-items: center;\r\n    background-color: #17191a;\r\n}\r\n  \r\n.ne-s-logout-container .ne-s-logotype-item {\r\n    position: absolute;\r\n    max-width: 110px;\r\n    top: 20px;\r\n    left: 100px;\r\n    display: flex;\r\n    align-items: center;\r\n}\r\n\r\n.ne-s-logotype-item img {\r\n    width: 110px;\r\n}\r\n\r\n.ne-s-logotype-item span {\r\n    margin-left: 15px;\r\n}\r\n  \r\n.ne-s-logout-container .ne-s-logout-page {\r\n    position: relative;\r\n    text-align: right;\r\n    background-color: #212426;\r\n    color: white !important;\r\n}\r\n\r\n.ne-s-logout-page > h2 {\r\n    font-size: 32px;\r\n    line-height: 36px;\r\n    margin: 20px 32px 0;\r\n}\r\n\r\n.ne-s-logout-page > h3 {\r\n    font-size: 14px;\r\n    font-style: italic;\r\n    line-height: 36px;\r\n    margin: 0 32px 20px;\r\n}\r\n\r\n.ne-s-logout-page h2,.ne-s-logout-page > h3 {\r\n    text-align: left;\r\n    font-weight: 500;\r\n}\r\n\r\n.ne-s-logout-page >.ne-s-netent-button {\r\n    margin: 0 32px 32px 32px;\r\n    min-width: 120px;\r\n    border-radius: 30px;\r\n    background-color: #68d122;\r\n    color: white;\r\n    text-transform: uppercase;\r\n    font-weight: 600;\r\n}\r\n\r\n.ne-s-logout-page .ne-s-logout-toolbar {\r\n    background-color: #68d122 !important;\r\n    color: white !important;\r\n    height: 64px;\r\n    display: flex;\r\n    box-sizing: border-box;\r\n    padding: 0 16px;\r\n    width: 100%;\r\n    flex-direction: row;\r\n    align-items: center;\r\n    white-space: nowrap;\r\n}\r\n\r\n/* .ne-s-logotype-item {\r\n    width: 100%;\r\n    height: 100vh;\r\n    min-height: 700px;\r\n    overflow: hidden;\r\n    position: relative;\r\n    background: url('../../assets/images/hero-startpage-better-gaming-1920x1000.jpg') no-repeat center;\r\n    background-size: cover;\r\n} */"]}