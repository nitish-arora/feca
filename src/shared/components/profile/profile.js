import React from "react";
import PropTypes from "prop-types";

import ExitToApp from "@material-ui/icons/ExitToApp";
import AccountBox from "@material-ui/icons/AccountBox";
import Tooltip from "@material-ui/core/Tooltip";
import { makeStyles, createStyles } from "@material-ui/core/styles";

import "./profile.css";
import * as Constants from "./../../constants/profile";

/**
 * Component to display the current user's profile.
 *
 * user: the user to display
 * onLogout: emits on logout
 *
 * ---js---
 * user: {
 *  username: 'USER_NAME',
 *  role: 'ROLE'
 * }
 *
 * handleLogout() {
 *  // handle logout
 * }
 *
 * ---html---
 * <Profile user={user} onLogout={() => handleLogout()} />
 */

//themeing the material ui components
const useStyles = makeStyles(theme =>
  createStyles({
    icon: {
      margin: theme.spacing(1),
      fontSize: 18
    }
  })
);

function Profile(props) {
  const classes = useStyles();
  return (
    <div className="ne-s-profile-container">
      <div className="profile-toolbar">
        <Tooltip
          disableFocusListener
          disableTouchListener
          title={Constants.ACCOUNT}
        >
          <div className="account-logo">
            <AccountBox className={classes.icon} />
          </div>
        </Tooltip>
        <div className="ne-s-text-name">{`${props.user.username}`}</div>
        <Tooltip
          disableFocusListener
          disableTouchListener
          title={Constants.LOGOUT}
        >
          <div className="ne-s-logout-btn" onClick={props.onLogout}>
            <ExitToApp className={classes.icon} />
          </div>
        </Tooltip>
      </div>
    </div>
  );
}

/** setting validations on props received from parent component */
Profile.propTypes = {
  user: PropTypes.shape({
    username: PropTypes.string,
    role: PropTypes.string
  }),
  onLogout: PropTypes.func
};

export default Profile;
