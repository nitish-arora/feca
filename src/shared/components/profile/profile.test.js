import React from 'react';
import Profile from './profile';
import renderer from 'react-test-renderer';
import './profile.css'

const handleLogout = function() {
  console.log('gdgsdg');
};
const user = {
    "username":"freja.bear@netent.com",
    "role":"netentinternal"
};

test('should match snapshot with hidden options', () => {
  const component = renderer.create(
    <Profile user={user} onLogout={handleLogout()} />
  );
  const selectBox = component.root.findAll(
    element => element.props.className === 'ne-s-logout-btn'
  );
  expect(component.toJSON()).toMatchSnapshot();
  expect(selectBox).toHaveLength(1);
});
