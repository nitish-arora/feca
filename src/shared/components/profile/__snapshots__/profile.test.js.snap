// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot with hidden options 1`] = `
<div
  className="ne-s-profile-container"
>
  <div
    className="profile-toolbar"
  >
    <div
      aria-describedby={null}
      className="account-logo"
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      title="Account"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root makeStyles-icon-1"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M3 5v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2H5c-1.11 0-2 .9-2 2zm12 4c0 1.66-1.34 3-3 3s-3-1.34-3-3 1.34-3 3-3 3 1.34 3 3zm-9 8c0-2 4-3.1 6-3.1s6 1.1 6 3.1v1H6v-1z"
        />
      </svg>
    </div>
    <div
      className="ne-s-text-name"
    >
      freja.bear@netent.com
    </div>
    <div
      aria-describedby={null}
      className="ne-s-logout-btn"
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      title="Logout"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root makeStyles-icon-1"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M10.09 15.59L11.5 17l5-5-5-5-1.41 1.41L12.67 11H3v2h9.67l-2.58 2.59zM19 3H5c-1.11 0-2 .9-2 2v4h2V5h14v14H5v-4H3v4c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
        />
      </svg>
    </div>
  </div>
</div>
`;
